### ISSUE TITLE
Supporting moddability in games made with bevy_fluent 

### ISSUE THREAD

**Issue by TheLeonsver1** [kgv/bevy_fluent#8]
So my game's structure might be something like this:
```
assets
    mods
        [mod_name]
            data
                events
            localization
            scripts
        [mod_name]
            data
                events
            localization
            scripts
```
And since it might be hard to use `std::fs` to scan folders on some platforms, I'd very much like it, if `FluentServer` could get a `load_folder()` function that would receive a language identifier (for example `en-US`) and when I'll give it the path `mods` it will load all localization files that are inside my mods folder and are related to the en-US localization, something like bevy's `AssetServer` with its recursive `load_folder()`.

I think it would require `bevy_fluent`'s `.ron` files to have an extension other than `.ron`. It might not be needed in a bevy 0.5 version if it'll happen, but it would be better in a 0.6 version probably because of this https://github.com/bevyengine/bevy/pull/2292 I think

Thanks again for the crate!

**Comment by kgv** [comment]
Initially, I had plans to support this functionality ([see implicit section](https://github.com/kgv/bevy_fluent/blob/63aabdd1429aaccaca74f8105476b2e43e02a3af/DOCUMENTATION.en-US.md#implicit)).
But then I gave it up because I couldn't find a good reason why I needed it. 
However, dynamically loadable modules/plugins are where this functionality are needed. 
But in the implementation, I ran into a number of difficulties.
According to preliminary estimates, I think it will be implemented in one form or another.

I am phrasing my thoughts more precisely after the weekend.

**Comment by TheLeonsver1** [comment]
yeah, don't worry about it :) I don't mean to pressure you, I'm currently thinking about pausing my moddable-game project anyway, so this issue is more of a question if it might ever be added haha. It could definitely wait for after 0.6 since it might be easier to do then hopefully.
